ros1_bridge/__init__.py

```diff
@@ -562,17 +562,23 @@ def determine_common_services(
     for rule in mapping_rules:
         for ros1_srv in ros1_srvs:
             for ros2_srv in ros2_srvs:
+
+                pair = (ros1_srv, ros2_srv)
+
+                if pair in pairs:
+                    continue
+
```

Comment:
nitpick: eliminate the newlines in line 565, 567, 570. Otherwise LGTM.

Reply:
Done